addons/account_edi_peppol/data/peppol_ubl_templates.xml

```diff
@@ -0,0 +1,198 @@
+                    <cbc:Name t-esc="partner.name"/>
+                    <cbc:Telephone t-if="partner.phone" t-esc="partner.phone"/>
+                    <cbc:ElectronicMail t-if="partner.email" t-esc="partner.email"/>
+                  </cac:Contact>
+            </cac:Party>
+        </template>
+
+        <!--
+        Render an invoice's line to be added to the UBL xml document.
+        -->
+        <template id="export_peppol_invoice_line">
+            <cac:InvoiceLine
+                xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+                xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+                xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+                <cbc:ID t-esc="invoice_line_id"/>
+                <t t-set="invoice_line_id" t-value="invoice_line_id + 1"></t>
+                <cbc:Note t-if="line.discount">Discount (<t t-esc='line.discount'/> %)</cbc:Note>
+                <cbc:InvoicedQuantity unitCode="ZZ"><t t-esc="line.quantity"/></cbc:InvoicedQuantity>
+                <cbc:LineExtensionAmount
+                    t-att-currencyID="invoice.currency_id.name"
+                    t-esc="format_monetary(convert_monetary(line.price_subtotal, line.currency_id or invoice.currency_id))"/>
+                <cac:Item>
+                    <cbc:Description t-esc="line.name.replace('\n', ', ')"/>
+                    <cbc:Name t-esc="line.product_id.name"/>
+                    <cac:SellersItemIdentification t-if="line.product_id.default_code">
+                        <cbc:ID t-esc="line.product_id.default_code"/>
+                    </cac:SellersItemIdentification>
+                    <t t-if="line.tax_ids">
+                        <t t-foreach="line.tax_ids" t-as="tax">
+                            <cac:ClassifiedTaxCategory>
+                                <cbc:ID>S</cbc:ID>
+                                <cbc:Percent t-esc="tax.amount" />
+                                <cac:TaxScheme>
+                                    <cbc:ID>VAT</cbc:ID>
+                                </cac:TaxScheme>
+                            </cac:ClassifiedTaxCategory>
+                        </t>
+                    </t>
+                    <t t-else="" >
+                        <cac:ClassifiedTaxCategory>
+                            <cbc:ID>Z</cbc:ID>
+                            <cbc:Percent>0</cbc:Percent>
+                            <cac:TaxScheme>
+                                <cbc:ID>VAT</cbc:ID>
+                            </cac:TaxScheme>
+                        </cac:ClassifiedTaxCategory>
+                    </t>
+                </cac:Item>
+                <cac:Price>
+                    <cbc:PriceAmount
+                        t-att-currencyID="invoice.currency_id.name"
+                        t-esc="format_monetary(convert_monetary(line.price_subtotal, line.currency_id or invoice.currency_id))"/>
```

Comment:
t-esc="format_monetary(convert_monetary(line.price_subtotal, invoice.currency_id))"/>

Reply:
Also, I think you should replace `line.price_subtotal` by `line.price_unit`

Reply:
> The price of an item, exclusive of VAT, after subtracting item price discount. The Item net price has to be equal with the Item gross price less the Item price discount, if they are both provided.